modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -636,18 +629,19 @@ class CV_EXPORTS_W VideoCapture
     implementation if multiple are available: e.g. cv::CAP_FFMPEG or cv::CAP_IMAGES or cv::CAP_DSHOW.
     @sa The list of supported API backends cv::VideoCaptureAPIs
     */
-    CV_WRAP VideoCapture(const String& filename, int apiPreference);
+    CV_WRAP VideoCapture(const String& filename, int apiPreference = CAP_ANY);
 
     /** @overload
     @brief  Open a camera for video capturing
 
-    @param index camera_id + domain_offset (CAP_*) id of the video capturing device to open. To open default camera using default backend just pass 0.
-    Use a `domain_offset` to enforce a specific reader implementation if multiple are available like cv::CAP_FFMPEG or cv::CAP_IMAGES or cv::CAP_DSHOW.
-    e.g. to open Camera 1 using the MS Media Foundation API use `index = 1 + cv::CAP_MSMF`
+    @param index id of the video capturing device to open. To open default camera using default backend just pass 0.
+    (Usage of camera_id + domain_offset (CAP_*) is valid.in this case apiPreference is ignored)
```

Comment:
@alalek what is your opinion?

(Usage of camera_id + domain_offset (CAP_*) is valid.in this case apiPreference is ignored)

Reply:
Using "domain offset" should be limited to default `apiPreference = CAP_ANY` case only.